test/index.js

```diff
@@ -162,477 +140,262 @@ describe('koa-body', () => {
       .attach('thirdField', 'README.md')
       .attach('thirdField', 'package.json')
       .expect(201)
-      .end( (err, res) => {
-        if (err) return done(err);
+      .then((res) => {
         res.body.user.names.should.be.an.Array().and.have.lengthOf(2);
         res.body.user.names[0].should.equal('John');
         res.body.user.names[1].should.equal('Paul');
         res.body._files.firstField.should.be.an.Object();
         res.body._files.firstField.name.should.equal('package.json');
-        should(fs.statSync(res.body._files.firstField.path)).be.ok;
+        should(fs.statSync(res.body._files.firstField.path)).be.ok();
```

Comment:
Different styles of using `should`, we should settle on one or the other. I'm still a little unclear on what will and will not have the `.should`  property, whereas `should(expr)` is pretty clear. When it works though, the property version is nicer to read.

Reply:
One of the reasons I dislike should.js is because it messes with prototypes to make assertion properties available. Some objects from `sinon` and others don't get monkey patched so we have to use this other style.

I'd like to move completely to using `should(x)...` instead of relying on the properties or even get rid of should completely.